sources/ElkArte/Modules/Random/Display.php

```diff
@@ -74,6 +76,23 @@ public static function followup_message_list($messages)
 		$context['follow_ups'] = followupTopics($messages, self::$includeUnapproved);
 	}
 
+	/**
+	 * Show likes.
+	 *
+	 * @param int[] $messages
+	 */
+	public static function load_likes($messages)
+	{
+		global $modSettings, $context, $txt;
+
+		if (!empty($modSettings['likes_enabled']) && !empty($messages))
```

Comment:
I we check for likes enabled in the section where load_likes is set as an integration function this could be simplified to 
`if (!empty($messages))`